### PR TITLE
Make ~idempotent by moving import sort; add tests

### LINKED ISSUE #56: gofumpt returns different results when run multiple times
A bit bizare bug.

I have this file (with tabs)

```
package main

import (
        "os"

        "io"
)
```

When I run `gofumpt -w foo.go`, I get

```
package main

import (
        "os"
        "io"
)
```

and when I run for a second time, I get

```
package main

import (
        "io"
        "os"
)
```

The tests did not catch this, since in the testscripts, there is always

```
gofumpt -w foo.go .
```

which actually visits the file *twice*, once as a `foo.go` and a second time when traversing the directory `.`.

A simple hacky fix is to format each file twice.

### PR DESCRIPTION
**Pull request by karel-3d**
Commented out one import test that's not currently working

Fixes #56

### CODE REVIEW

**Comment by karel-3d** [comment]
"~idempotent" because I know of code that returns differently on multiple rounds; it's the code that's currently commented out :)

File: gofmt.go [hunk 1]
```diff
@@ -108,8 +108,6 @@ func processFile(filename string, in io.Reader, out io.Writer, stdin bool) error
 		}
 	}
 
-	ast.SortImports(fileSet, file)
```
Comment (karel-3d) [inline gofmt.go:111]: not sure if I should have left it there and copy it, or move it. Seems like both works and I can't find an edgecase where it doesn't.

**Comment by karel-3d** [comment]
#59 for the commented-out test

**Comment by mvdan** [comment]
Done, see a91da47f375c4e5a5c8ad84e3f888fb042f80ad0.

**Comment by karel-3d** [comment]
Thx! Seems like a cleaner solution

**Pull request closed without merging**